src/v3/src/transformer/redirect/redirectTransformer.ts

```diff
@@ -63,6 +62,7 @@ export const redirectTransformer = (
   const { label, name } = appInfo;
   const appName = label ?? name;
```

Comment:
When you debug, can you break here and see if appInfo has any data or possibly check the transaction object and see if there is any information in the context that indicates the app name? 

I only ask because with this, I thought maybe the bookmark name would show up as the app name but maybe not? 🤔

Reply:
appInfo did not have any data.  I looked at the transaction object and there is no app data like name or anything

Reply:
![image](https://user-images.githubusercontent.com/107433508/199584797-41a84c09-10ef-47f1-94ee-67d842504559.png)

Reply:
Awesome! So this is expected! Great! Thank you so much!